# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1386015500 18000
#      Mon Dec 02 15:18:20 2013 -0500
# Branch stable
# Node ID c0c61619917ee8eda8fc41a3a4542adbab1717e0
# Parent  62f94ef938b48a1e477fa7bc3e781aecb61e21b4
update library version numbers for upcoming release

* liboctave/Makefile.am (liboctave_current): Bump to 2.
(liboctave_revision, liboctave_age): Set to 0.
* liboctinterp/Makefile.am (liboctinterp_current): Bump to 2.
(liboctinterp_revision, liboctinterp_age): Set to 0.
* liboctgui/Makefile.am (liboctgui_current, liboctgui_revision,
liboctgui_age): Set to 0 for first official release with GUI library.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -182,18 +182,18 @@ liboctinterp_la_LIBADD = \
   parse-tree/libparse-tree.la \
   parse-tree/libparser.la \
   corefcn/libcorefcn.la \
   corefcn/libtex_parser.la \
   $(top_builddir)/liboctave/liboctave.la \
   $(LIBOCTINTERP_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
-liboctinterp_current = 1
-liboctinterp_revision = 1
+liboctinterp_current = 2
+liboctinterp_revision = 0
 liboctinterp_age = 0
 
 liboctinterp_version_info = $(liboctinterp_current):$(liboctinterp_revision):$(liboctinterp_age)
 
 liboctinterp_la_LDFLAGS = \
   -version-info $(liboctinterp_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   -bindir $(bindir) \
diff --git a/liboctave/Makefile.am b/liboctave/Makefile.am
--- a/liboctave/Makefile.am
+++ b/liboctave/Makefile.am
@@ -99,18 +99,18 @@ liboctave_la_LIBADD = \
   numeric/libnumeric.la \
   operators/liboperators.la \
   system/libsystem.la \
   util/libutil.la \
   $(top_builddir)/libgnu/libgnu.la \
   $(LIBOCTAVE_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
-liboctave_current = 1
-liboctave_revision = 1
+liboctave_current = 2
+liboctave_revision = 0
 liboctave_age = 0
 
 liboctave_version_info = $(liboctave_current):$(liboctave_revision):$(liboctave_age)
 
 liboctave_la_LDFLAGS = \
   -version-info $(liboctave_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   @XTRA_CRUFT_SH_LDFLAGS@ \
